examples/example-parent.yaml

```
@@ -0,0 +1,6 @@
+--- 
+StatePaths:
```

following the naming convention from DSF, this should be `statePaths`.
I'd even consider calling it `stateFiles` instead

```suggestion
stateFiles:
```